cli/config.go

```diff
@@ -97,7 +97,7 @@ func registerFlags(c *Config, f *flag.FlagSet) {
 	f.StringVar(&c.Status, "status", "good", "Status of the certificate: good, revoked, unknown")
 	f.StringVar(&c.Reason, "reason", "0", "Reason code for revocation")
 	f.StringVar(&c.RevokedAt, "revoked-at", "now", "Date of revocation (YYYY-MM-DD)")
-	f.Int64Var(&c.Interval, "interval", int64(4*helpers.OneDay), "Interval between OCSP updates, in seconds (default: 4 days)")
+	f.DurationVar(&c.Interval, "interval", 4*helpers.OneDay, "Interval between OCSP updates (default: 96h)")
```

Comment:
@jsha this may affect the current cli ocsp signer behavior, would you take a look?